src/amo/utils/server.js

```diff
@@ -59,3 +67,23 @@ export const viewFrontendVersionHandler = ({
     });
   };
 };
+
+export const viewHeartbeatHandler = ({
+  _config = config,
+  _fetch = fetch,
+}: ViewHeartbeatHandlerParams = {}): ExpressHandler => {
+  const apiURL = `${_config.get('apiHost')}${_config.get(
+    'apiPath',
+  )}${_config.get('apiVersion')}/site/?disable_caching`;
+  return async (req: typeof $Request, res: typeof $Response) => {
+    const response = await _fetch(apiURL);
+    if (response.status !== 200) {
+      res.sendStatus(400);
+    } else {
+      const siteStatus = await response.json();
+      res.json({ siteStatus });
```

Comment:
Do we really need to send the content of the response?

We could simplify the logic here, and only return:

```js
res.status(response.status).end(response.status === 200 ? 'ok' : 'ko');
```

Reply:
I guess I was thinking that at some point maybe `siteStatus` might include other information about the site's status, that might be relevant, but a) that's actually unlikely and b) the calling code probably only cares about the status code, so yeah, we can simplify it like that.